TWLight/views.py

```diff
@@ -124,6 +124,7 @@ def get_context_data(self, **kwargs):
         context["no_of_languages"] = (
             Partner.objects.values("languages").distinct().count()
         )
+        context["all_languages"] = Language.objects.values_list("language", flat=True)
```

Comment:
Exclude current language here (and rename to something like alternate_languages)